DESCRIPTION

```diff
@@ -28,13 +28,13 @@ Depends:
     R (>= 4.0)
 Imports:
     checkmate (>= 2.1.0),
+    cli (>= 3.4.0),
```

Comment:
Needs to be added to `Config/Needs/verdepcheck: mllg/checkmate, r-lib/lifecycle, r-lib/rlang,
Config/Needs/verdepcheck` as well (and `.pre-commit-config.yaml`

Reply:
added 👍